app/models/package.scala

```diff
@@ -5,7 +5,7 @@ package object models {
   import play.api.Logger
   import play.api.Play.current
 
-  lazy val db = {
+  def db = {
```

Comment:
This change means that a new database instance and datasource pool will be created every time db is used in a model, doesn't it?  Was this change needed to get tests running or were you doing this to set up some restructuring for better tests?

I was able to run the tests with it still being a lazy val.  I merged your changes minus the removal of lazy val's to the master branch.

I personally don't like the db method or lazy val db that much.  I am used to doing dependency injection for database pools.  I was thinking about trying that out on this code base.  Play has a general hook for doing DI and there are multiple adapters to various DI systems.  I was thinking about trying out Guice or Scaldi with this app.  One of the problems I was thinking about though that I don't know how to solve, would be testing without setting up a remote database.  I would prefer to run unit tests against something runs in memory only.  That would mean using another database other then postgres for unit tests.  I haven't figured out how to abstract the way the slick library is used for multiple databases though.  It seems like slick code imports database specific drivers at compile time.